<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP-GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RunCreate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск сборки выбранного образа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Архитектура</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_arch">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x86_64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>i586</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Каталог с мета-профилем</string>
    </property>
    <widget class="QLineEdit" name="DirProfiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>/usr/share/mkimage-profiles</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DirProfiles">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Обзор</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_git">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Скачать последнюю версию  мета-профиля</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор образа для сборки</string>
    </property>
    <widget class="QComboBox" name="comboBox_ListDistro">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор файла настроек apt.conf, если не указано, используются системные настройки.</string>
    </property>
    <widget class="QLineEdit" name="AptConf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AptConf">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Обзор</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>721</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор каталога  для генерируемого профиля и сборки, если не указан - автовыбор системой.</string>
    </property>
    <widget class="QPushButton" name="pushButton_BuildDir">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Обзор</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BuildDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ReMake">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>721</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сборка образа с внесёнными изменениями  из выбранного каталога (без обращения к мета-профилю)</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_Check">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим проверки без сборки образа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Settings">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RedSet">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить настройки</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_mirror">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Sisyphus-mirror</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
